Overview.src.html

```diff
@@ -850,90 +852,128 @@ <h2 class="no-num no-toc">
         <p>
           The <code>change</code> event is fired during the execution of the
           <span>monitor the list of available presentation displays</span>
-          algorithm when the <dfn>presentation display availability</dfn>
-          changes. It is fired at the <span><code>Availability</code></span>
+          algorithm. It is fired at the <span><code>Availability</code></span>
           object, using the <code>ChangeEvent</code> interface. The
           <code>Availability</code> object's <code>value</code> attribute is
           updated to the boolean value that the algorithm determined.
         </p>
-        <p>
-          In order to satisfy the <a href=
-          "https://github.com/w3c/presentation-api/blob/gh-pages/uc-req.md#req08-power-saving-friendly">
-          power saving non-functional requirement</a>, the user agent must keep
-          track of the number of <code>Availability</code> objects requested
-          through the <code>getAvailability()</code> method. The UA may also
-          keep track of whether the page holding the <code>Availability</code>
-          object is in the foreground. Using this information, implementation
-          specific discovery of <span title="presentation-display">presentation
-          displays</span> can be resumed or suspended, in order to save power.
-        </p>
-        <p>
-          The user agent must keep a <dfn>list of available presentation
-          displays</dfn>. According to the number of alive
-          <code>Availability</code> objects, the user agent must also keep the
-          list up to date by running the algorithm to <span>monitor the list of
-          available presentation displays</span>.
-        </p>
-        <p>
-          The user agent might not support continuous availability monitoring
-          due to platform or power consumption restrictions, for example. In
-          this case the <span>Promise</span> returned by
-          <code>getAvailability()</code> is <span data-anolis-spec="promguide"
-          title="resolve-reject">rejected</span> and the algorithm to
-          <span>monitor the list of available presentation displays</span> will
-          only run as part of the <span title="startSession">session
-          start</span> algorithm.
-        </p>
         <section>
-          <h5>
-            Monitoring the list of available presentation displays
-          </h5>
+          <h4>
```

Comment:
Nit: Don't you want to have a section for each header to make it accessible from TOC?